common/src/commonMain/kotlin/org/hisp/dhis/common/screens/RadioButtonScreen.kt

```diff
@@ -81,5 +93,9 @@ fun RadioButtonScreen() {
         RadioButtonBlock(Orientation.VERTICAL, radioButtonDataItemsVertical, selectedItemVertical) {
             selectedItemVertical = it
         }
+        SubTitle("Horizontal Age Field Helper")
+        AgeFieldHelper(Orientation.HORIZONTAL, AgeFieldHelperValues.YEARS.value) {
```

Comment:
What do you think about creating a InputAge screen to set this component and then we can use it form the future InputAge component?

Reply:
done!